packages/docmanager-extension/src/index.ts

```diff
@@ -378,15 +378,15 @@ function addCommands(
           return;
         }
         docManager.services.contents.get(path, { content: false }).then(
-          args => {
+          item => {
             // exists
-            return commands.execute(CommandIDs.open, { path: path });
+            return commands.execute('filebrowser:navigate', { path: path });
```

Comment:
In my mind, it was cleaner to move this open from path command over to the filebrowser, rather than keep here it here in the doc manager. The odd part about it being here is that the filebrowser will call back into the doc manager to open the thing if it is a file, so it seems weird to call into the filebrowser and have the filebrowser call back here. That's not a hard dependency cycle, like was discussed in this PR. Rather, it's a logical command execution chain that seems weird.

Reply:
I was thinking the same thing when I wrote my [original commit](https://github.com/jupyterlab/jupyterlab/pull/6561/commits/2223fa980dbad7dae0f3a50d832da76fd8855bb1). As @telamonian noted, it is a lot more of a change in the codebase for the same functionality. Do you think I should pull in that restructured code again?